### ISSUE TITLE
Changing the amplifier Id

### ISSUE THREAD

**Issue by vchamberland** [labstreaminglayer/App-EGIAmpServer#7]
We have no problem running the EGIAmpServer connector, even two at the time since we have two EGI NetAmp 400 in a hyperscanning setup.

However, since both amplifier Id are set to the default value 0, it is rather confusing to have two 'EGI NetAmp 0' in the LSL pipeline (e.g., StreamViewer, LabRecorder, etc.).

If I change the default Id value to something other than 0, then EGIAmpServer is no longer able to link to the amplifiers.

This is maybe a dumb question, but how to change the default Amplifier Id value to something like 1 or 2?

**Comment by mamoran66** [comment]
I don't see where the ID can be assigned so I suspect it is assigned automatically if two amps with the same IP are detected

**Comment by vchamberland** [comment]
That might be the case, but we have two amps so two distinct IP.

You can see the field "Amplifier Id" in the Recording Settings section:

<img width="168" alt="EGIampserver" src="https://github.com/labstreaminglayer/App-EGIAmpServer/assets/71802740/70e58ecb-c856-46a8-bdf0-9f715b51aed3">

**Comment by mamoran66** [comment]
Right, but in the SDK I do not see a "setID" call thus I suspect it is done automatically. It would be great to see the code that is activated when a user chooses a different ID in the recording settings UI

**Comment by tstenner** [comment]
The amplifierId is passed to the commands sent to the configured EGI server application. IIRC it's possible to connect multiple amplifiers to a single mac and they will have different amplifierIds. Independent amplifiers will have the amplifierId 0.
